src/solution_0d3d703e.py

```diff
@@ -0,0 +1,78 @@
+import json
+import numpy as np
+import sys
+
+
+translations_dict = {1: 5, 2: 6, 3: 4, 4: 3, 5: 1, 6: 3, 8: 9, 9: 8}
+
+
+def read_json(file_path):
```

Comment:
Could this be moved to a module maybe as im sure it would be common across all solutions we have

Reply:
I will do that - makes sense - yes